packages/storage/src/service.ts

```diff
@@ -129,104 +221,23 @@ export class StorageService {
     }
   }
 
-  /**
-   * Returns a firebaseStorage.Reference for the given path in the default
-   * bucket.
-   */
-  ref(path?: string): Reference {
-    function validator(path: unknown): void {
-      if (typeof path !== 'string') {
-        throw 'Path is not a string.';
-      }
-      if (/^[A-Za-z]+:\/\//.test(path as string)) {
-        throw 'Expected child path but got a URL, use refFromURL instead.';
-      }
-    }
-    args.validate('ref', [args.stringSpec(validator, true)], arguments);
-    if (this.bucket_ == null) {
-      throw new Error('No Storage Bucket defined in Firebase Options.');
-    }
-
-    const ref = new Reference(this, this.bucket_);
-    if (path != null) {
-      return ref.child(path);
-    } else {
-      return ref;
-    }
-  }
-
-  /**
-   * Returns a firebaseStorage.Reference object for the given absolute URL,
-   * which must be a gs:// or http[s]:// URL.
-   */
-  refFromURL(url: string): Reference {
-    function validator(p: unknown): void {
-      if (typeof p !== 'string') {
-        throw 'Path is not a string.';
-      }
-      if (!/^[A-Za-z]+:\/\//.test(p as string)) {
-        throw 'Expected full URL but got a child path, use ref instead.';
-      }
-      try {
-        Location.makeFromUrl(p as string);
-      } catch (e) {
-        throw 'Expected valid full URL but got an invalid one.';
-      }
-    }
-    args.validate('refFromURL', [args.stringSpec(validator, false)], arguments);
-    return new Reference(this, url);
-  }
-
   get maxUploadRetryTime(): number {
     return this.maxUploadRetryTime_;
   }
 
-  setMaxUploadRetryTime(time: number): void {
-    args.validate(
-      'setMaxUploadRetryTime',
-      [args.nonNegativeNumberSpec()],
-      arguments
-    );
+  set maxUploadRetryTime(time: number) {
     this.maxUploadRetryTime_ = time;
   }
 
   get maxOperationRetryTime(): number {
     return this.maxOperationRetryTime_;
```

Comment:
The setters and getters perform no validation and have no side-effects. We can just make this a property.

Reply:
Done.

Reply:
Done.